src/App.css

```diff
@@ -471,15 +469,6 @@ Footer {
   max-width: 130px;
 }
 
-@media only screen and (min-width: 720px) {
-  .logo {
```

Comment:
The logo has to be able to move outside the list element. I have updated the logo sizing in my PR